### ISSUE TITLE
CMAS Portal Accessibility testing violation #1

### ISSUE THREAD

**Issue by pkegbeto** [carbon-design-system/carbon#677]
### Widget alternate text missing

| __Violations__ |__Classification__ | __Description__ |__Image__|
|----------------|--------------------|-----------------|-------
| [haac.g1128.MobileAriaImgAlt](https://aat.w3ibm.mybluemix.net/token/ce020aa2-f761-46a8-8925-2deeda4c9f69/a763761a-383e-4445-8e73-dbf86b61c844/latest/doc/w3/help/en-US/idhi_accessibility_check_g1128.html)| Carbon Components |```<svg aria-label="Forward" width="8" viewBox="0 0 8 12" role="img" name="chevron--right" height="12" fill-rule="evenodd" class="bx--pagination__button-icon">``` |![](https://media.github.ibm.com/user/31397/files/f7ac0fe6-2de9-11e8-8eb5-ce445ea17370) |
||Carbon Components|```<svg aria-label="Backward" width="8" viewBox="0 0 8 12" role="img" name="chevron--left" height="12" fill-rule="evenodd" class="bx--pagination__button-icon">```|![](https://media.github.ibm.com/user/31397/files/d24d1490-2df1-11e8-9bd4-149d33ffab1c) |
|| Carbon Components |```<svg aria-label="open list of options" width="10" viewBox="0 0 10 5" role="img" name="caret--down" height="5" fill-rule="evenodd" class="bx--select__arrow">``` |![](https://media.github.ibm.com/user/31397/files/8740c1d8-2def-11e8-94c9-d18fa945f909) |
|| Carbon Components | ```<svg style="width: 100%;" aria-label="open and close list of options" width="32" viewBox="0 0 32 32" role="img" name="user" height="32" fill-rule="evenodd" class="bx--overflow-menu__icon">``` |![](https://media.github.ibm.com/user/31397/files/a0d6e1b2-2df0-11e8-815c-5c30fc41727a)|
|| Carbon Components |```<svg aria-label="display sub navigation items" width="12" viewBox="0 0 12 8" role="img" name="chevron--down" height="8" fill-rule="evenodd" class="left-nav-list__item-icon bx--interior-left-nav__icon">``` | ![](https://media.github.ibm.com/user/31397/files/97fefa14-2df2-11e8-8973-e67a7e481b8c)|
||Carbon Components|```<svg aria-label="search" width="16" viewBox="0 0 16 16" role="img" name="search--glyph" height="16" fill-rule="evenodd" class="bx--search-magnifier">```|![](https://media.github.ibm.com/user/31397/files/64a2cf6e-2df3-11e8-9f3f-8b5222f06997) |

**Comment by alisonjoseph** [comment]
The latest pagination component should have all the correct a11y roles on the buttons wrapping the svgs. Feel free to re-open if you are still seeing issues.
